contrib/docker/artisanqueue.conf

```diff
@@ -0,0 +1,18 @@
+[group:artisanqueue]
+programs=artisanqueue
+priority=99
+
+[program:artisanqueue]
+process_name=%(program_name)s
+command=/usr/local/bin/php /app/artisan queue:work
+user=application
+startsecs = 0
+autostart = true
+autorestart = true
+startsecs = 10
+startretries = 5
+stdout_logfile=/dev/stdout
```

Comment:
Not sure how supervisors works here - if you inspect the docker logs does the worker come through?

Reply:
@intentionally-left-nil : AFAICT this work, e.g. if I interact with the app I get logs like this in the stdout of the container:
```
83.65.162.102 - - [05/Sep/2023:08:04:22 +0200] "POST /i/delete HTTP/1.1" 200 43 "https://pixelfed.apps.fedi.at/i/web/timeline/local" "Mozilla/5.0 (X11; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/117.0"
[php-fpm:access] 127.0.0.1 -  05/Sep/2023:08:04:22 +0200 "POST /index.php" 200 /app/public/index.php 90.266 2048 66.47%
  2023-09-05 06:04:22 App\Jobs\StatusPipeline\StatusDelete ........... RUNNING
  2023-09-05 06:04:22 App\Jobs\StatusPipeline\StatusDelete ..... 220.76ms DONE
  2023-09-05 06:04:22 App\Jobs\MediaPipeline\MediaDeletePipeline ..... RUNNING
  2023-09-05 06:04:22 App\Jobs\MediaPipeline\MediaDeletePipeline  40.82ms DONE
  ```

Reply:
thanks for checking!